configuration/pih/scripts/global/zl.js

```diff
@@ -433,4 +433,158 @@ function updateLastCheckboxRequired(containerSelector, transRequiredMsgId, requi
 
   // Add the updateLastCheckboxRequired function to the beforeSubmit array to call it before form submission.
   beforeSubmit.push(updateLastCheckbox);
-}
\ No newline at end of file
+}
+patientDossierEditAlert();
+saveSelectedLocation();
+//Creating this function to alert the clinician when He/She wants to modify
+// Patient info which is not in the same location
+function patientDossierEditAlert() {
+  jq(document).ready(function () {
+
+    //Track the previously saved loggeInUserLocation/Clinician from the localStorage
+    let loggedInUserLocation = localStorage.getItem("location");
+    let arraysValue=[]
+ 
+    jq("li.float-left:eq(2)").click(function (e) {
+      // Getting the patient location
+      jq('fieldset.programstatus-fieldset').each(function () {
+       // put all data from the fieldset in a array
+        let firstPTag = jq(this).find('p.programstatus-field:nth-child(2):first');
+        let locations = firstPTag.text().trim();
+        arraysValue.push(locations)
+     
+    });
+    //Remove duplication
+    let uniqueArrays = arraysValue.filter((array, index, self) => {
+  
+      return self.indexOf(array) === index;
+    });
+      //get the location of the patient
+      var patientLocation=uniqueArrays[1];
+
+      if (loggedInUserLocation != patientLocation) {
```

Comment:
I'm seeing a problem where the logged in user set to null and the patient location appears to be an empty array, and these aren't equal, so the error message is rendering... this should be null/empty safe.  there is another problem that just occurred to me that I'm seeing now as well...